app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -279,79 +284,134 @@ class RevisionCardFragmentTest {
   }
 
   @Test
-  fun testRevisionCardTestActivity_fractionSubtopicId1_checkReturnToTopicButtonIsDisplayedSuccessfully() { // ktlint-disable max-line-length
+  fun testRevisionCardTestActivity_fractionSubtopicId0_checkPreviousNavCardIsNotDisplayed() {
     launch<RevisionCardActivity>(
       createRevisionCardActivityIntent(
         context,
         profileId.internalId,
         FRACTIONS_TOPIC_ID,
-        SUBTOPIC_TOPIC_ID
+        SUBTOPIC_TOPIC_ID_0
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
 
-      onView(withId(R.id.revision_card_return_button))
-        .check(matches(withText(R.string.return_to_topic)))
+      onView(withId(R.id.previous_navigation_card))
```

Comment:
Should the next one be displayed? Suggest adding a check for it and renaming the test to be "checkOnlyPrevNavCardIsNotDisplayed".

Reply:
done